code/controllers/subsystem/blackbox.dm

```diff
@@ -29,6 +35,11 @@ SUBSYSTEM_DEF(blackbox)
 	var/datum/DBQuery/query_record_playercount = SSdbcore.NewQuery("INSERT INTO [format_table_name("legacy_population")] (playercount, admincount, time, server_ip, server_port) VALUES ([playercount], [admincount], '[SQLtime()]', INET_ATON('[world.internet_address]'), '[world.port]')")
 	query_record_playercount.Execute()
 
+	if(config.use_exp_tracking)
+		if((triggertime < 0) || (world.time > (triggertime +3000)))	//server maint fires once at roundstart then once every 10 minutes. a 5 min check skips the first fire. The <0 is midnight rollover check
+			update_exp(10,0)

```

Comment:
Nope. Wont work. Server maint having a wait of 6 causes it to fire far too often for this. You changed it from 10 minutes making it useless for what I needed.